app/src/main/java/com/framgia/forder/screen/createshop/CreateshopPresenter.java

```diff
@@ -1,13 +1,20 @@
 package com.framgia.forder.screen.createshop;
 
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
 import android.text.TextUtils;
+import android.util.Base64;
 import com.framgia.forder.data.model.User;
 import com.framgia.forder.data.source.ShopRepository;
 import com.framgia.forder.data.source.UserRepository;
 import com.framgia.forder.data.source.remote.api.error.BaseException;
 import com.framgia.forder.data.source.remote.api.error.SafetyError;
 import com.framgia.forder.data.source.remote.api.request.RegisterShopRequest;
 import com.framgia.forder.data.source.remote.api.response.RegisterShopResponse;
+import com.framgia.forder.data.source.remote.api.response.ShopManagementResponse;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.source.remote.api.response.ShopManagementResponse. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports